eyeshade/controllers/owners.js

```diff
@@ -464,15 +464,47 @@ v1.getWallet = {
         }
       }
 
-      entries = await settlements.find({ owner: owner }, { sort: { timestamp: -1 }, limit: 1 })
-      entry = entries && entries[0]
+      entries = await publishers.find({ owner: owner })
+      summary = await settlements.group(
```

Comment:
https://docs.mongodb.com/manual/reference/method/db.collection.group/
> Deprecated since version 3.4: Mongodb 3.4

We are running version 3.4.9 in prod I think

Reply:
_typical. just typical._ i now have to find what the equivalent command is.